t/app/controller/admin/bodies.t

```diff
@@ -2,8 +2,6 @@ package FixMyStreet::Cobrand::Tester;
 
 use parent 'FixMyStreet::Cobrand::Default';
 
-sub enable_category_groups { 1 }
```

Comment:
Don't need this whole package now.

Reply:
Sadly it's required so we can both have a cobrand for setting the features on and for the `MAPIT_TYPES` override to work so the `Mock::MapIt` call works. If you use any other cobrand then the internal `MAPIT_TYPES` setting of the cobrand overrides the override.

Reply:
Surely the 'default' cobrand would be fine as it doesn't set one, like the UK ones? But fine, doesn't matter :)